_includes/releases/v23.1/v23.1.0.md

```diff
@@ -0,0 +1,513 @@
+   </td>
+   <td><p>The default <a href="https://www.cockroachlabs.com/docs/v23.1/configure-replication-zones.html#gc-ttlseconds">GC TTL value</a> in 23.1.0 is being reduced from 25 hrs to 4 hrs for new clusters.
+   <p>
+This change is being made to improve read performance, storage utilization, and cluster stability in high write traffic scenarios.
+<a href="https://www.cockroachlabs.com/docs/v23.1/manage-a-backup-schedule.html">Scheduled backups</a> will not be affected by this change as <a href="https://www.cockroachlabs.com/docs/v23.1/architecture/storage-layer#protected-timestamps">protected timestamps</a> will ensure data isn't garbage-collected until it has been backed up. <a href="https://www.cockroachlabs.com/docs/v23.1/create-changefeed.html">Changefeeds</a> will also not be affected. If you want a 25-hour or larger GC TTL value (for example, to support <a href="https://www.cockroachlabs.com/docs/v23.1/as-of-system-time.html">AS OF SYSTEM TIME</a> queries that go further back in time), you can explicitly set GC TTL to the desired value. 
+<p>
+This change will only apply to new clusters. Existing clusters will retain the 25-hour default when upgrading, unless you have previously overridden it with an explicit value, in which case, that value will be retained. Backups taken from clusters running versions prior to v23.1 will similarly retain the GC TTL configured when the backup was taken.
+   </td>
+  </tr>
+  </tbody>
+</table>
+
+<h4 id="v23-1-0-backward-incompatible-changes">Backward-incompatible changes</h4>
+
+Before [upgrading to CockroachDB v23.1](../v23.1/upgrade-cockroach-version.html), be sure to review the following backward-incompatible changes, as well as key cluster setting changes, and adjust your deployment as necessary.
+
+- Replaced the `cdc_prev()` [function](../v23.1/functions-and-operators.html) in favor of a `cdc_prev` tuple. This is an incompatible change that may break [changefeeds](../v23.1/change-data-capture-overview.html) that use the previous `cdc_prev()` function. [#85177][#85177]
+- [`SHOW RANGES FOR TABLE`](../v23.1/show-ranges.html) now includes rows for all indexes that support the table. Prior to this change, `SHOW RANGES FOR TABLE foo` was an alias for `SHOW RANGES FOR INDEX foo@primary`. This was causing confusion, as it would miss data for secondary indexes. It is still possible to filter to just the primary index using `SHOW RANGES FOR INDEX foo@primary`. The statement output now also includes the index name. [#93545][#93545]
+- CockroachDB now supports sharing storage ranges across multiple indexes/tables. As a result, there is no longer a guarantee that there is at most one SQL object (e.g., table/index/sequence/materialized view) per storage range.  Therefore, the columns `table_id`, `database_name`, `schema_name`, `table_name` and `index_name` in `crdb_internal.ranges` and `.ranges_no_leases` have become nonsensical: a range cannot be attributed to a single table/index anymore.  As a result:
```

Comment:
This bullet point and the points below need to clarify the relationship with the new cluster setting wrt the deprecated behavior.

NB: even though the _default_ behavior is compatible with v22.2, I still would like this to be mentioned in the backward-incompatible changes, to stimulate customers to adapt their automation

Reply:
> This bullet point and the points below need to clarify the relationship with the new cluster setting wrt the deprecated behavior.

I've updated this so that it says 

> ... cannot be attributed to a single table/index anymore. For the v23.1 release, the default behavior of [`SHOW RANGES`](../v23.1/show-ranges.html) is retained, but users should consider setting the [cluster setting `sql.show_ranges_deprecated_behavior.enabled` to `false`](../v23.1/cluster-settings.html#setting-sql-show-ranges-deprecated-behavior-enabled). This will have the following effects that will become the defaults in a future release: